.github/workflows/upgrade_downgrade_test_backups_manual.yml

```diff
@@ -126,33 +126,36 @@ jobs:
         sudo deluser mysql
         sudo rm -rf /var/lib/mysql
         sudo rm -rf /etc/mysql
-
         # Install MySQL 8.0
         ####
-        ## Pin the MySQL version at 8.0.29 as Vitess 13.0 does not have the fix to support
-        ## backups of 8.0.30+ and no backport to v13.0 is currently planned.
-        ##   See: https://github.com/vitessio/vitess/pull/10847
+        ## Temporarily pin the MySQL version at 8.0.29 as Vitess 14.0.1 does not have the fix to support
```

Comment:
To be pedantic we should change the comment to reflect reality too. Will approve to unblock you.